diff --git a/targets/ath79-generic b/targets/ath79-generic
index 7a5140c9..4e9a3ad7 100644
--- a/targets/ath79-generic
+++ b/targets/ath79-generic
@@ -391,7 +391,7 @@ device('tp-link-archer-c2-v3', 'tplink_archer-c2-v3', {

 device('tp-link-archer-c25-v1', 'tplink_archer-c25-v1', {
        packages = ATH10K_PACKAGES_SMALLBUFFERS_QCA9887,
-       broken = true, -- OOM with 5GHz enabled in most environments
+       broken = false, -- ts: false - OOM with 5GHz enabled in most environments
        class = 'tiny', -- 64M ath9k + ath10k
 })

