diff --git a/targets/ar71xx-generic b/targets/ath79-generic
index df7dbd9..fe30d3b 100644
--- a/targets/ath79-generic
+++ b/targets/ath79-generic
@@ -355,7 +355,7 @@ device('tp-link-archer-c7-v5', 'archer-c7-v5', {
 
 device('tp-link-archer-c25-v1', 'archer-c25-v1', {
 	packages = ATH10K_PACKAGES_QCA9887,
-	broken = true, -- OOM with 5GHz enabled in most environments
+	broken = false, -- ts: set to false; OOM with 5GHz enabled in most environments
 	class = 'tiny', -- 64M ath9k + ath10k
 })
 
