diff --git a/target/linux/x86/config-4.14 b/target/linux/x86/config-4.14
index 355a8cb..305fc57 100644
--- a/openwrt/target/linux/x86/config-5.10
+++ b/openwrt/target/linux/x86/config-5.10
@@ -5,6 +5,7 @@
 # CONFIG_ADVANTECH_WDT is not set
 # CONFIG_ALIM1535_WDT is not set
 # CONFIG_ALIX is not set
+CONFIG_PATA_ATIIXP=y
 CONFIG_AMD_NB=y
 CONFIG_ARCH_CLOCKSOURCE_DATA=y
 CONFIG_ARCH_DEFCONFIG="arch/x86/configs/i386_defconfig"
